test/platform/riot/coap_channel_federated_test/sender/Makefile

```diff
@@ -0,0 +1,41 @@
+# name of your application
+APPLICATION = lf-coap-federated-sender
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../../../../../../RIOT
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+# Enable Unity
+USEMODULE += Unity
```

Comment:
The unit test framework? Is it used for anything?

Reply:
No this was a leftover, I fixed it :)